app/models/users/student.rb

```diff
@@ -12,4 +14,11 @@ def after_sign_in_path
   def self.is_student?(data)
     Student.where(email: data["email"]).first
   end
+
+  private
+
+  def create_student_playlist
+    student = Student.last
```

Comment:
Instead of using Student.last to get the last created student, would it be better to just put `self` on line 22?